tests/e2e/actor_pubsub/actor_pubsub_test.go

```diff
@@ -0,0 +1,357 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package actor_pubsub_e2e
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/dapr/dapr/tests/e2e/utils"
+	kube "github.com/dapr/dapr/tests/platforms/kubernetes"
+	"github.com/dapr/dapr/tests/runner"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/ratelimit"
+)
+
+var tr *runner.TestRunner
+
+const (
+	numberOfMessagesToPublish = 60
+	publishRateLimitRPS       = 25
+	defaultActorType          = "myActorType"
```

Comment:
Please, use an actor type that is specific to this test to avoid conflicting with other test apps.